config/environment.js

```diff
@@ -103,6 +103,8 @@ module.exports = function (environment) {
       terms: 'https://docs.travis-ci.com/legal/terms-of-service/',
       tutorial: 'https://docs.travis-ci.com/user/tutorial/',
       twitter: 'https://twitter.com/travisci',
+      pardotHost: 'https://info.travis-ci.com',
+      pardotForm: '/l/845883/2020-02-03/257j'
```

Comment:
Is there a reason to split the URL into pieces here? Are there any plans on other integrations with Pardot at the moment? If not, then I'd like to suggest keeping it as a single URL for now, until we see the pattern how to organize the Pardot urls better. What do you think?